### PR TITLE
Determine if time should be 24 hours from locale

### PR DESCRIPTION
**Pull request by Tyilo**
This only changes the default value.
I have also added the --no-military option to
force showing 12 hour time.

### CODE REVIEW

**Comment by jcrowgey** [comment]
Please use the `locale` module.

File: gcalcli/argparsers.py [hunk 1]
```diff
@@ -103,9 +114,13 @@ def get_output_parser(parents=[]):
     output_parser.add_argument(
             '--width', '-w', default=10, dest='cal_width', type=validwidth,
             help='Set output width')
+    has_24_hours = locale_has_24_hours()
     output_parser.add_argument(
-            '--military', action='store_true', default=False,
+            '--military', action='store_true', default=has_24_hours,
             help='Use 24 hour display')
+    output_parser.add_argument(
+            '--no-military', action='store_false', default=has_24_hours,
```
Comment (jcrowgey) [inline gcalcli/argparsers.py]: I think you're going to need a `dest` for this option if you want to integrate it with the current application code which is going to be looking for `military`.

Comment (Tyilo) [reply gcalcli/argparsers.py]: Fixed

**Comment by Tyilo** [comment]
> Please use the `locale` module.

As in #458, I don't think this is possible.

**Comment by jcrowgey** [comment]
Ok, what I am I missing in this one?

```shell
$ python
Python 2.7.16 (default, Apr  6 2019, 01:42:57) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import time
>>> import locale
>>> locale.setlocale(locale.LC_TIME, 'en_US.UTF-8')
'en_US.UTF-8'
>>> time.strftime(locale.nl_langinfo(locale.T_FMT), time.gmtime())
'07:49:13 PM'
>>> locale.setlocale(locale.LC_TIME, 'nl_NL.UTF-8')
'nl_NL.UTF-8'
>>> time.strftime(locale.nl_langinfo(locale.T_FMT), time.gmtime())
'19:49:30'
>>> 
```

**Comment by Tyilo** [comment]
Ah, you're right, updated.

**Comment by jcrowgey** [comment]
Tested locally and I'm happy.  In terms of curiosities, gcalcli now accepts both `--no-miliitary` and `--military` and will go with whichever one is last.  I suppose that's reasonable.

Thanks for this!

**Pull request merged**